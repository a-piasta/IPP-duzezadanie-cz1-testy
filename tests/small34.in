extendRoute;50;vydkkndhuqroqbarzoppkd
repairRoad;lnjhsxflisyhlzwoyotlizixhelu;vydkkndhuqroqbarzoppkd;-2
getRouteDescription;124
extendRoute;50;vydkkndhuqroqbarzoppkd
repairRoad;runeezyyapoiosknkiehmgyy;wtzpcwpxtlj;-6
newRoute;821;wtzpcwpxtlj;ucyyapvuaefudly
extendRoute;821;lklxramdzsdlsgwoiumklj
getRouteDescription;821
newRoute;650;gxatvukpjqviotidjsucejooov;wtzpcwpxtlj
newRoute;741;ccxrypvccpjkizhkx;ccxrypvccpjkizhkx
extendRoute;50;lnjhsxflisyhlzwoyotlizixhelu
addRoad;lklxramdzsdlsgwoiumklj;gsoxaykevvljkssbqqf;24;0
addRoad;runeezyyapoiosknkiehmgyy;lnjhsxflisyhlzwoyotlizixhelu;0;1
repairRoad;lklxramdzsdlsgwoiumklj;lnjhsxflisyhlzwoyotlizixhelu;12
getRouteDescription;821
newRoute;540;lklxramdzsdlsgwoiumklj;uzecnamtlfnnjwdamab
newRoute;389;vydkkndhuqroqbarzoppkd;gsoxaykevvljkssbqqf
addRoad;gxatvukpjqviotidjsucejooov;uzecnamtlfnnjwdamab;15;5
newRoute;520;lklxramdzsdlsgwoiumklj;gsoxaykevvljkssbqqf
repairRoad;ucyyapvuaefudly;wtzpcwpxtlj;3
getRouteDescription;650
removeRoad;gxatvukpjqviotidjsucejooov;uzecnamtlfnnjwdamab
addRoad;wtzpcwpxtlj;lnjhsxflisyhlzwoyotlizixhelu;6;4
extendRoute;389;gxatvukpjqviotidjsucejooov
addRoad;vydkkndhuqroqbarzoppkd;runeezyyapoiosknkiehmgyy;7;-15
removeRoad;uzecnamtlfnnjwdamab;gxatvukpjqviotidjsucejooov
addRoad;wtzpcwpxtlj;lnjhsxflisyhlzwoyotlizixhelu;28;-6
removeRoad;lnjhsxflisyhlzwoyotlizixhelu;runeezyyapoiosknkiehmgyy
repairRoad;vydkkndhuqroqbarzoppkd;lklxramdzsdlsgwoiumklj;-1
removeRoad;lklxramdzsdlsgwoiumklj;gsoxaykevvljkssbqqf
newRoute;902;lklxramdzsdlsgwoiumklj;lklxramdzsdlsgwoiumklj
newRoute;192;gsoxaykevvljkssbqqf;runeezyyapoiosknkiehmgyy
getRouteDescription;192
removeRoad;uzecnamtlfnnjwdamab;gxatvukpjqviotidjsucejooov
addRoad;gxatvukpjqviotidjsucejooov;uzecnamtlfnnjwdamab;6;0
getRouteDescription;821
getRouteDescription;540
extendRoute;540;ucyyapvuaefudly
repairRoad;gsoxaykevvljkssbqqf;vydkkndhuqroqbarzoppkd;7
getRouteDescription;192
extendRoute;821;lklxramdzsdlsgwoiumklj
extendRoute;192;lnjhsxflisyhlzwoyotlizixhelu
getRouteDescription;902
addRoad;ccxrypvccpjkizhkx;ucyyapvuaefudly;9;12
removeRoad;gsoxaykevvljkssbqqf;lklxramdzsdlsgwoiumklj
extendRoute;520;runeezyyapoiosknkiehmgyy
repairRoad;ccxrypvccpjkizhkx;ccxrypvccpjkizhkx;9
repairRoad;lnjhsxflisyhlzwoyotlizixhelu;vydkkndhuqroqbarzoppkd;11
repairRoad;lklxramdzsdlsgwoiumklj;gsoxaykevvljkssbqqf;-8
addRoad;gsoxaykevvljkssbqqf;runeezyyapoiosknkiehmgyy;17;-8
getRouteDescription;389
extendRoute;50;lklxramdzsdlsgwoiumklj
addRoad;uzecnamtlfnnjwdamab;ccxrypvccpjkizhkx;27;-1
getRouteDescription;821
removeRoad;ucyyapvuaefudly;ccxrypvccpjkizhkx
removeRoad;uzecnamtlfnnjwdamab;gxatvukpjqviotidjsucejooov
extendRoute;821;runeezyyapoiosknkiehmgyy
newRoute;111;wtzpcwpxtlj;gsoxaykevvljkssbqqf
