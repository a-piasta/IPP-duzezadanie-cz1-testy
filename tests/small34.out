1: NIE
2: NIE
3: 
4: NIE
5: NIE
6: NIE
7: NIE
8: 
9: NIE
10: NIE
11: NIE
12: NIE
13: NIE
14: NIE
15: 
16: NIE
17: NIE
18: TAK
19: NIE
20: NIE
21: 
22: TAK
23: TAK
24: NIE
25: TAK
26: NIE
27: NIE
28: NIE
29: NIE
30: NIE
31: NIE
32: NIE
33: 
34: NIE
35: NIE
36: 
37: 
38: NIE
39: NIE
40: 
41: NIE
42: NIE
43: 
44: TAK
45: NIE
46: NIE
47: NIE
48: NIE
49: NIE
50: TAK
51: 
52: NIE
53: TAK
54: 
55: TAK
56: NIE
57: NIE
58: NIE
